Content.Server/Roles/AntagonistRole.cs

```diff
@@ -0,0 +1,50 @@
+using Content.Shared.Roles;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+
+namespace Content.Server.Roles
+{
+    public class AntagonistRole : Role
+    {
+        /// <summary>
+        ///     Path to antagonist alert sound.
+        ///     TODO: Traitor sound will be a default one, cause there is no other sounds right now.
+        /// </summary>
+        protected virtual SoundSpecifier AntagonistAlert { get; } = new SoundPathSpecifier("/Audio/Ambience/Antag/traitor_start.ogg");
+
+        public AntagPrototype Prototype { get; }
+
+        public override string Name { get; }
+
+        public override bool Antagonist { get; }
+
+        /// <summary>
+        /// .ctor
+        /// </summary>
+        /// <param name="mind">A mind (player)</param>
+        /// <param name="antagPrototype">Antagonist prototype</param>
+        public AntagonistRole(Mind.Mind mind, AntagPrototype antagPrototype) : base(mind)
+        {
+            Prototype = antagPrototype;
+            Name = Loc.GetString(antagPrototype.Name);
+            Antagonist = antagPrototype.Antagonist;
+        }
+
+        public override void Greet()
+        {
+            base.Greet();
+
+            // Alert a player about antagonist role with a sound notification
+            var entMgr = IoCManager.Resolve<IEntityManager>();
+            entMgr.EntitySysManager.TryGetEntitySystem(out SharedAudioSystem? audio);
+            if (audio != null && Mind.Session != null)
+            {
+                audio.PlayGlobal(
+                    AntagonistAlert,
+                    Filter.Empty().AddPlayer(Mind.Session),
+                    false,
+                    AudioParams.Default);
+            }
+        }
```

Comment:
i think you should just move the greet sound to whatever antag system/comp that doesn't have something like that yet (i think just zombies) and remove this

Reply:
The purpose of it is to not make a bunch of the same code in different antagonist classes.

Reply:
playing the greet sound is something so minor that it doesn't warrant adding more code to something that will be removed sooner or later like the role class and its inheritors, just repeat for now

Reply:
Moved to sound notifications to System components.